object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionChecker.java

```diff
@@ -63,6 +69,12 @@ public Properties getMessagesProperties() {
     messages.setProperty(
         "finalizer.invocation.invalid",
         "This finalizer cannot be invoked, because the following methods have not been called: %s\n");
+    messages.setProperty(
+        "ensuresvarargs.annotation.invalid",
+        "@EnsuresCalledMethodsVarArgs cannot be written on a non-varargs method");
+    messages.setProperty(
+        "ensuresvarargs.unverified",
+        "@EnsuresCalledMethodsVarArgs cannot be verified yet.  Please suppress this warning.");
```

Comment:
"Please check that the implementation of the method actually does call the given methods on the varargs parameters by hand, and then suppress the warning."